packages/indexer-common/src/indexer-management/monitor.ts

```diff
@@ -336,7 +336,7 @@ export class NetworkMonitor {
     }
     let subgraphs: Subgraph[] = []
     const queryProgress = {
-      lastCreatedAt: 0,
+      lastId: 0,
```

Comment:
The network subgraph's schema is using type `ID` for ids, which is an alias for `string`.
(see https://semiotic-ai.slack.com/archives/C032EDF093K/p1713996760318369?thread_ts=1713983989.035979&cid=C032EDF093K)

```suggestion
      lastId: '',
```